test/e2e/storage/persistent_volumes-local.go

```diff
@@ -132,6 +132,42 @@ var _ = framework.KubeDescribe("[Volume] PersistentVolumes-local [Feature:LocalP
 			f.TestContainerOutput("pod writes to PV", podSpec, 0, []string{testVol.hostDir})
 		})
 	})
+
+	Context("when two pods request one prebound PVC", func() {
+		var (
+			testVol *localTestVolume
+			node    *v1.Node
+		)
+
+		BeforeEach(func() {
+			// Choose the first node
+			node = &config.nodes[0]
+		})
+
+		AfterEach(func() {
+			cleanupLocalVolume(config, testVol)
+			testVol = nil
+		})
+
+		It("should be able to mount volume, write from pod1, and read from pod2", func() {
+			By("Initializing test volume")
+			testVol = setupLocalVolume(config, node)
+
+			By("Creating local PVC and PV")
+			createLocalPVCPV(config, testVol)
+
+			By("Creating a pod to write to the PV")
+			testFilePath := filepath.Join("/mnt/volume1", testFile)
+			cmd := fmt.Sprintf("echo %s > %s; cat %s", testVol.hostDir, testFilePath, testFilePath)
```

Comment:
Can you put the read and write commands into wrappers, since many of the tests are doing the same thing?